doc/Language/iterating.pod6

```diff
@@ -4,136 +4,321 @@
 
 =SUBTITLE Functionalities available for visiting all items in a complex data structure
 
-=head1 The C<Iterator> and C<Iterable> roles
+Similar to other many programming languages, Raku makes a fine distinction among
+the terms I<iteration>, I<iterable>, and I<iterator>. Familiarizing yourself
```

Comment:
iteration is nothing like iterable, Why would someone *not* want to make that distinction?